apps/docsite/src/Welcome.mdx

```diff
@@ -0,0 +1,57 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Welcome" />
+
+# Fluent UI Contributors Packages
+
+Welcome to the documentation site for Fluent UI Contributors packages! These packages are published under the `@fluentui-contrib/` scope and provide additional functionality to enhance your Fluent UI experience.
+
+## Available Packages
+
+Explore the various packages available to extend your Fluent UI capabilities:
+
+- [**houdini-utils**](/docs/packages-houdini-utils--docs): Utilities for CSS Houdini.
```

Comment:
Currently, the list is manually compiled, but we might explore generating it automatically from Storybook stories.

Reply:
create issue to track this pls

Reply:
https://github.com/microsoft/fluentui-contrib/issues/246